sop-core/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.6.0.0
+
+* Generalise the type signatures of cata- and anamorphisms
```

Comment:
Is this a breaking change? What could break?

Reply:
Since what's being generalized is top level definitions, and not class members, I would imagine this is not a breaking change (under the assumption that the generalization is indeed a strict generalization).